android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -190,6 +215,7 @@ class RNPlayerView(
         // is responsible for destroying the player in the end.
         playerView.player = null
         playerView.onDestroy()
+        (playerView.parent as? ViewGroup)?.removeView(playerView)
```

Comment:
That was just for debugging, please remove

Reply:
Address in 554967836fc4e69ceaed9cca7bd9b446cee85ad8